pkgs/servers/authelia/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib, fetchFromGitHub, buildGoModule, installShellFiles, fetchurl }:
+
+buildGoModule rec {
```

Comment:
if you do use `finalAttrs` here, `ui` can be overriden safely without issue for `postPatch` and the rest, I think ; up to you :).

Reply:
Maybe it has no support to the new format yet.

Reply:
It has no support for the new format, indeed!